app/component/itinerary/ItineraryPage.js

```diff
@@ -717,6 +818,18 @@ export default function ItineraryPage(props, context) {
     altStates[PLANTYPE.BIKETRANSIT][0].plan,
   ]);
 
+  // merge the main plan and the scooter plan into one
+  useEffect(() => {
+    if (
+      state.loading === LOADSTATE.DONE &&
+      scooterState.loading === LOADSTATE.DONE &&
+      scooterState?.plan?.edges?.length > 0
+    ) {
+      const plan = mergeScooterTransitPlan(scooterState.plan, state.plan);
+      setCombinedRentalState({ plan });
+    }
+  }, [scooterState.plan, state.plan]);
+
```

Comment:
There is a minor issue: when user hits earlier button, the new set is merged with standard itineraries, not with scooter-combined itineraries. Same with later button if arrive by option is set.

Reply:
Canot duplicate. Most likely 'user error'.